src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1

```diff
@@ -133,7 +133,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Initial Release'
+        ReleaseNotes = 'Added Teams to Office Data Connector, Bug Fixes and updated documentation.'
```

Comment:
same here, please revert it back to previous release notes

Reply:
fixed